ballerina/jwt_validator.bal

```diff
@@ -57,7 +57,7 @@ public type ValidatorSignatureConfig record {|
         cache:CacheConfig cacheConfig?;
         ClientConfiguration clientConfig = {};
     |} jwksConfig?;
-    string certFile?;
+    string|crypto:PublicKey certFile?;
```

Comment:
Shall we update the documentation as well? Saying that we can give a file path or a `crypto:PublicKey` record as the certificate?

Reply:
Fixed with [857b997c9142b2a14d59a545823326dc49b86191](https://github.com/ballerina-platform/module-ballerina-jwt/pull/1229/commits/857b997c9142b2a14d59a545823326dc49b86191)